client/app/telecom/telephony/alias/administration/terminate/telecom-telephony-alias-administration-terminate.html

```diff
@@ -90,16 +90,30 @@ <h1 data-translate="telephony_alias_administration_terminate_title"></h1>
                                    data-translate-attr="{ 'placeholder': 'telephony_alias_administration_terminate_details' }">
                         </textarea>
                     </div>
-                    <button type="button"
-                            class="btn btn-default"
+                    <div class="form-group mt-4">
+                        <hr/>
+                        <telephony-bulk-action
+                            data-service-type="alias"
+                            data-billing-account="{{ AliasTerminateCtrl.bulkDatas.billingAccount }}"
+                            data-service-name="{{ AliasTerminateCtrl.bulkDatas.serviceName }}"
```

Comment:
You can use AliasTerminateCtrl.number.billingAccount / serviceName instead, and purge bulkDatas